### PR TITLE
Update to the new documentation URL

### PR DESCRIPTION
**Pull request by kneth**
## What, How & Why?

The documentation now lives on a new URL

## ☑️ ToDos

* [ ] 📝 Changelog entry
* [ ] 📝 `Compatibility` label is updated or copied from previous entry
* [ ] 📝 Update `COMPATIBILITY.md`
* [ ] 🚦 Tests
* [ ] 📦 Updated internal package version in consuming `package.json`s (if updating internal packages)
* [ ] 📱 Check the React Native/other sample apps work if necessary
* [ ] 💥 `Breaking` label has been applied or is not necessary

### CODE REVIEW

File: packages/realm-react/README.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ For a full fledged example, check out [our templates](https://github.com/realm/r
 ## Realm Hooks
 
 ### useRealm
-Returns the instance of the [`Realm`](https://www.mongodb.com/docs/realm-sdks/js/latest/Realm.html) configured by `createRealmContext` and the `RealmProvider`.  The following is an example of how to use this Hook to make a write transaction callback for a component.
+Returns the instance of the [`Realm`](https://www.mongodb.com/docs/realm-sdks/js/latest/classes/Realm.html) configured by `createRealmContext` and the `RealmProvider`.  The following is an example of how to use this Hook to make a write transaction callback for a component.
```
Comment (kraenhansen) [inline packages/realm-react/README.md:122]: This is giving me a 403 Forbidden.

File: packages/realm-react/README.md [hunk 2]
```diff
@@ -146,7 +146,7 @@ const Component = ({item}) => {
 
 ### useQuery
 
-Returns [`Realm.Results`](https://www.mongodb.com/docs/realm-sdks/js/latest/Realm.Results.html) from a given type. This Hook will update on any changes to any Object in the Collection and return an empty array if the Collection is empty.
+Returns [`Realm.Results`](https://www.mongodb.com/docs/realm-sdks/js/latest/classes/Realm.Results.html) from a given type. This Hook will update on any changes to any Object in the Collection and return an empty array if the Collection is empty.
```
Comment (kraenhansen) [inline packages/realm-react/README.md:149]: This too is giving me a 403 Forbidden.

File: packages/realm/README.md [hunk 3]
```diff
@@ -14,27 +14,27 @@ Realm is a mobile database that runs directly inside phones, tablets or wearable
...
+The [Atlas Device SDKs](https://www.mongodb.com/docs/atlas/device-sdks/) are a collection of language and platform specific SDKs, each with a suite of app development tools optimized for data access and persistence on mobile and edge devices. Use the SDKs to build data-driven mobile, edge, web, desktop, and IoT apps.
 
 It might help to think of the Realm database as the persistance layer of the Atlas Device SDKs.
 
 ## Features
 
 * **Mobile-first:** Realm is the first database built from the ground up to run directly inside phones, tablets and wearables.
-* **Simple:** Data is directly [exposed as objects](https://www.mongodb.com/docs/realm/sdk/node/realm-files/) and [queryable by code](https://www.mongodb.com/docs/realm/sdk/node/crud/query-data/), removing the need for ORM's riddled with performance & maintenance issues.
+* **Simple:** Data is directly [exposed as objects](https://www.mongodb.com/docs/atlas/device-sdks/sdk/node/realm-files/) and [queryable by code](https://www.mongodb.com/docs/atlas/device-sdks/node/crud/query-data/), removing the need for ORM's riddled with performance & maintenance issues.
```
Comment (kraenhansen) [inline packages/realm/README.md:24]: The https://www.mongodb.com/docs/atlas/device-sdks/node/crud/query-data/ gives a 404.

**Review by kraenhansen — COMMENTED** [review]
Please double check the links, at least 3 of them leads me nowhere 🙈

File: packages/realm-react/README.md (hunk 1 above)
Comment (kneth) [reply packages/realm-react/README.md:122]: Yes, something is wrong in our API documentation. Currently https://www.mongodb.com/docs/realm-sdks/js/latest/classes/export_-1.html is generated. This PR is assuming we fix the API documentation.

File: packages/realm-react/README.md (hunk 2 above)
Comment (kneth) [reply packages/realm-react/README.md:149]: See above

File: packages/realm-react/README.md (hunk 1 above)
Comment (elle-j) [reply packages/realm-react/README.md:122]: Our [reference docs](https://www.mongodb.com/docs/realm-sdks/js/latest/modules.html#Realm) (especially for the `Realm` class) actually seem to have been affected by the export refactor 🤔

File: packages/realm/README.md (hunk 3 above)
Comment (kneth) [reply packages/realm/README.md]: fixed

File: packages/realm-react/README.md (hunk 1 above)
Comment (kneth) [reply packages/realm-react/README.md:122]: I'll try to fix it in a separate PR

**Review by elle-j — APPROVED** [review]
Thanks for making the updates!

File: packages/realm-react/README.md (hunk 1 above)
Comment (kraenhansen) [reply packages/realm-react/README.md:122]: > the export refactor

What PR are you referring to specifically?

Comment (elle-j) [reply packages/realm-react/README.md:122]: Likely the [Rollup one](https://github.com/realm/realm-js/pull/6492), where we started using `export import`.

Comment (kraenhansen) [reply packages/realm-react/README.md:122]: Yeah - I don't remember ever publishing docs from that change.

Comment (elle-j) [reply packages/realm-react/README.md:122]: The published docs are 12.6.1, and it seems that Rollup was removed in that release.

Comment (kneth) [reply packages/realm-react/README.md:122]: The v12.7.0 API docs looks fine

**Pull request merged**